ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-proxy-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-install-rbac
+  - chain: nutanix-provision-bastionhost
+  - ref: mirror-images-by-oc-adm-in-bastion
+  - ref: ipi-conf-proxy
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource-disconnected
+  - ref: set-sample-operator-disconnected
```

Comment:
I do not think we need the above two ref in cluster behind proxy, just need `ref: enable-qe-catalogsource`

Reply:
If possible, also pls help update `cucushift-installer-rehearse-nutanix-ipi-disconnected-provision` together.